pkg/ocm/idps.go

```diff
@@ -152,22 +152,29 @@ func getIDPListWithoutAuthURLSupport() []string {
 }
 
 // BuildOAuthURL builds the correct OAuthURL depending on the cluster type
-func BuildOAuthURL(cluster *cmv1.Cluster) (string, error) {
+func BuildOAuthURL(cluster *cmv1.Cluster, includePort bool) (string, error) {
```

Comment:
I see most types don't need to include port. What do you think of a check function kind like we have HasAuthURLSupport, so we don't need to include this bool

Reply:
good idea as it is more explicit, changed that